### DISCUSSION TITLE
Blue topaz  example lite

### DISCUSSION THREAD

**Discussion by cumany** [cumany/Blue-topaz-examples#32] [General]
这个是Blue topaz exmple的简化版，所谓简化版就是删减了大量不需要的辅助类插件，可以按需自己安装。简化版目前内置的插件只有20多个，轻装上阵，祝您用库愉快！
![4ad454b9-be8b-4603-aab4-96b7cfbe13e6](https://user-images.githubusercontent.com/42957010/173027434-7c6642d5-a4a4-4895-af3a-1da42a07acb9.png)

<br>
history:<br>
<em>20220630  <a href='https://github.com/cumany/Blue-topaz-examples/releases/tag/20220630-lite'>Blue topaz  example lite-0630</a>.</em><br>
<em>20220610  <a href='https://github.com/cumany/Blue-topaz-examples/releases/tag/20220610-lite'>Blue topaz  example lite-0610</a>.</em><br>

**Comment by ninjannette** [comment]
Hi! I'd like to make use of the icon pane on your right sidebar for my Vault. I would like the clickable icons to link to various notes that I frequently use. Since the example vault here is not in English, could you please share the specific markdown files, plugins and settings needed to set it up? Thank you so much!

**Reply by cumany** [reply]
The good news is that an English version is in the works.
The full version of the example vault has similar [examples](https://github.com/cumany/Blue-topaz-examples)
More specifically,, you need the Hotkeys for specific files plugin
and invoke the plugin command via the button
For example
````
```button
name mynote
type command
action Hotkeys for specific files: Open 50-Inbox/mynote
class circle_btn blank
```
````
